packages/byods/src/constants.ts

```diff
@@ -0,0 +1,4 @@
+export const BYODS_FILE = 'BYODS';
+export const USER_AGENT = 'BYoDS NodeJS SDK / 0.0.1';
```

Comment:
Is it better to have a version constant which can be concatenated into user agent?

Reply:
Done